src/skadi/sample.cc

```diff
@@ -212,7 +212,7 @@ class tile_data {
   bool reusable;
 
 public:
-  tile_data() : c(nullptr), index(0), reusable(false), data(nullptr) {
+  tile_data() : c(nullptr), index(TILE_COUNT), reusable(false), data(nullptr) {
```

Comment:
If I'm not mistaken - index=0 is actually a valid index. So it's better to fill it with invalid index value